tests/unit/conftest.py

```diff
@@ -1,14 +1,42 @@
 import pytest
 from fastapi.testclient import TestClient
+from fastapi_jwt_auth import AuthJWT
 
 from app.dependencies import RepositoryFactory
 from app.main import app
+from app.utils.auth import AuthUtils
 from tests.mocks import UserRepositoryMock
 
-MOCK_LINKING = {RepositoryFactory.user_repository: lambda: UserRepositoryMock()}
+MOCK_LINKING = {
+    RepositoryFactory.user_repository: lambda: UserRepositoryMock(),
+}
+
+
+@pytest.fixture
+def client_no_auth():
+    """Client sin autenticación (usado directamente como fixture)."""
+    app.dependency_overrides = MOCK_LINKING
+    return TestClient(app)
 
 
 @pytest.fixture(scope="class")
 def client(request):
+    """Client sin autenticación (puede usarse para probar accesos no autorizados)."""
     app.dependency_overrides = MOCK_LINKING
-    request.cls.client = TestClient(app)
+    request.cls.client_no_auth = TestClient(app)
+    return request.cls.client_no_auth

```

Comment:
Just return the client!
```suggestion
    return client
```